### PR TITLE
Defect sass deprecation

### PR DESCRIPTION
**Pull request by Mrjaco12**
* Update sass file names to fix deprecation warning.
* Added changelog for tracking releases.

### CODE REVIEW

**Comment by samcdavid** [comment]
@Mrjaco12 will you update `atomic_cms.gemspec` for version 0.2.4?

## Ruby Open Source Gem Release Checklist

### Pre Merge into Master

- [ ] Bump version numbers and dates in `gemspec` following [semver](http://semver.org/).

### Post Merge

- [ ] Checkout the master branch and pull the latest changes.
- [ ] Publish the gem with `rake publish`

**Comment by Mrjaco12** [comment]
I knew there was something.

**Comment by samcdavid** [comment]
That [checklist](https://github.com/spartansystems/spartan/blob/master/docs/Delivery/Open_Source_Release_Checklist.md) is in the spartan repo for future reference :smile:

**Comment by samcdavid** [comment]
:+1:

File: atomic_cms.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'atomic_cms'
-  s.version     = '0.2.3'
+  s.version     = '0.2.4'
   s.date        = '2015-06-19'
```
Comment (samcdavid) [inline atomic_cms.gemspec]: As a side note, it looks like the date isn't required per the [rubygems gemspec doc](http://guides.rubygems.org/specification-reference/)

Comment (Mrjaco12) [inline atomic_cms.gemspec]: Cool lets axe it and see what happens.

**Comment by handerson** [comment]
:+1:

**Pull request merged**